src/utils/barcode.ts

```diff
@@ -0,0 +1,73 @@
+import * as bwipjs from "bwip-js";
+import { sequenceS } from "fp-ts/lib/Apply";
+import { left, right, tryCatch2v } from "fp-ts/lib/Either";
+import {
+  fromEither,
+  taskEither,
+  TaskEither,
+  taskify
+} from "fp-ts/lib/TaskEither";
+import { NonEmptyString } from "italia-ts-commons/lib/strings";
+
+const BCID = NonEmptyString.decode(process.env.BARCODE_ALGO_ID).getOrElse(
```

Comment:
`process.env` should be accessed only by `config.ts` (otherwise that would be hard to track this global configuration values)
so this value must be moved there (I'd give it a more explicit name ie'. `BARCODE_ALGORITHM`